web/tests/popover_menus_data.test.js

```diff
@@ -136,7 +137,8 @@ function test(label, f) {
 }
 
 // Test functions
-test("my_message_all_actions", () => {
+test("my_message_all_actions", ({override_rewire}) => {
+    override_rewire(settings_data, "user_can_delete_any_message", noop);
```

Comment:
There is ongoing work to remove `override_rewire`, so we should try if we can avoid using them.

Reply:
I am not sure how I can avoid it easily?

Reply:
Can you override `is_user_in_group` function or just create "Everyone" group and set the setting to it.

Reply:
Creating the group is the preferred model, but this test seems to be mocking modules.

But probably what we want is a function like #30831 that creates all the system groups for a test? @afeefuddin @LaPulgaaa FYI, I think we can defer the issue for this PR.

Reply:
Yeah, I was able to avoid `override_rewire` by using `everyone` group.